vars/unlockNetwork.groovy

```diff
@@ -10,6 +10,6 @@ def call(String networkName) {
             '--network-name', networkName,
         ]
 
-        sh cmd.join(' ')
+        sh cmd.collect{ arg -> "'" + arg + "'" }.join(' ')
```

Comment:
Maybe we should move this fragment into separate function? It might be useful in the future.

Reply:
I've extracted it to `shellCommand` function. Also added escaping according to [this SO answer](https://unix.stackexchange.com/questions/187651/how-to-echo-single-quote-when-using-single-quote-to-wrap-special-characters-in/187654#187654).